hail/src/main/scala/is/hail/expr/ir/InferPType.scala

```diff
@@ -62,14 +62,17 @@ object InferPType {
         val et = coerce[PArray](a.pType).elementType
         PArray(et, a.pType.required)
       case ToSet(a) =>
-        val et = coerce[PArray](a.pType).elementType
+        val et = coerce[PIterable](a.pType).elementType
```

Comment:
Can we make it consistent between `et` and `elt`? Also, Does changing this to PIterable change any behavior?

Reply:
Doesn't change any behavior---just makes it convertible from a wider set of things right now.

Reply:
(I'll change to `elt` for consistency.)